recipes/poppler/all/conanfile.py

```diff
@@ -189,7 +189,8 @@ def _configure_cmake(self):
         self._cmake.definitions["ENABLE_DCTDECODER"] = self._dct_decoder
         self._cmake.definitions["USE_FLOAT"] = self.options.float
         self._cmake.definitions["RUN_GPERF_IF_PRESENT"] = False
-        self._cmake.definitions["ENABLE_RELOCATABLE"] = True
+        if self.settings.os == "Windows":
+            self._cmake.definitions["ENABLE_RELOCATABLE"] = self.options.shared
```

Comment:
Isn't this a little risky ? a static library can be linked into a shared library

Reply:
This remains possible, the new *owner* must provide its own `DllMain` entry point and make sure that poppler is initialized correctly.
In other words the same as when poppler is used in an ordinary executable.